src/time.jl

```diff
@@ -234,7 +234,7 @@ epc = Epoch("2018-12-01 16:22:19.123456789 GPS")
 ```
 """
 struct Epoch
-    # All days, seconds, and nanoseconds are stored internally in the TAI time scale, conversion to from TAI is done on intpu/output interacting.
+    # All days, seconds, and nanoseconds are stored internally in the TAI time scale, conversion to from TAI is done on intput/output interacting.
```

Comment:
This still has a typo in it `intput` instead of `input` 😄 

Fix the typo and this looks good to me. Appreciate the PR!

Reply:
lmao, I guess I'm not great at spelling either. fixed it.

Reply:
Thank you!